deploy/helm/sumologic/templates/metrics/collector/otelcol/opentelemetrycollector.yaml

```diff
@@ -129,6 +135,6 @@ spec:
     mountPath: /tmp
   - name: file-storage
     mountPath: /var/lib/storage/otc
-  config: |
+  config:
```

Comment:
Is it not important to keep the `|` here?

Reply:
this is because of the migration from api version `v1alpha` to `v1beta1`

Reply:
https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/crd-changelog.md#structured-configuration